`jspm` command fails with rollup depdency

When i installed the cli globally using `npm install -g jspm`.  I get the following error

```sh
jspm --version
file:///Users/jk/.nvm/versions/node/v22.20.0/lib/node_modules/jspm/dist/cli.js:3693
        throw new Error(
              ^

Error: Cannot find module @rollup/rollup-darwin-arm64. npm has a bug related to optional dependencies (https://github.com/npm/cli/issues/4828). Please try `npm i` again after removing both package-lock.json and node_modules directory.
    at requireWithFriendlyError (file:///Users/jk/.nvm/versions/node/v22.20.0/lib/node_modules/jspm/dist/cli.js:3693:15)
    at ../node_modules/rollup/dist/native.js (file:///Users/jk/.nvm/versions/node/v22.20.0/lib/node_modules/jspm/dist/cli.js:3699:147)
    at __require2 (file:///Users/jk/.nvm/versions/node/v22.20.0/lib/node_modules/jspm/dist/cli.js:18:50)
    at ../node_modules/rollup/dist/es/shared/parseAst.js (file:///Users/jk/.nvm/versions/node/v22.20.0/lib/node_modules/jspm/dist/cli.js:4659:29)
    at __init (file:///Users/jk/.nvm/versions/node/v22.20.0/lib/node_modules/jspm/dist/cli.js:15:56)
    at ../node_modules/rollup/dist/es/shared/node-entry.js (file:///Users/jk/.nvm/versions/node/v22.20.0/lib/node_modules/jspm/dist/cli.js:11655:5)
    at __init (file:///Users/jk/.nvm/versions/node/v22.20.0/lib/node_modules/jspm/dist/cli.js:15:56)
    at file:///Users/jk/.nvm/versions/node/v22.20.0/lib/node_modules/jspm/dist/cli.js:29553:1 {
  [cause]: Error: Cannot find module '@rollup/rollup-darwin-arm64'
  Require stack:
  - /Users/jk/.nvm/versions/node/v22.20.0/lib/node_modules/jspm/dist/cli.js
      at Function._resolveFilename (node:internal/modules/cjs/loader:1383:15)
      at defaultResolveImpl (node:internal/modules/cjs/loader:1025:19)
      at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1030:22)
      at Function._load (node:internal/modules/cjs/loader:1192:37)
      at TracingChannel.traceSync (node:diagnostics_channel:322:14)
      at wrapModuleLoad (node:internal/modules/cjs/loader:237:24)
      at Module.require (node:internal/modules/cjs/loader:1463:12)
      at require (node:internal/modules/helpers:147:16)
      at requireWithFriendlyError (file:///Users/jk/.nvm/versions/node/v22.20.0/lib/node_modules/jspm/dist/cli.js:3684:16)
      at ../node_modules/rollup/dist/native.js (file:///Users/jk/.nvm/versions/node/v22.20.0/lib/node_modules/jspm/dist/cli.js:3699:147) {
    code: 'MODULE_NOT_FOUND',
    requireStack: [
      '/Users/jk/.nvm/versions/node/v22.20.0/lib/node_modules/jspm/dist/cli.js'
    ]
  }
}
```

Maybe we need to check optional dependencies from the `rollup` version that is added for `jpm build`command

--- a/cli/chompfile.toml
+++ b/cli/chompfile.toml
@@ -67,7 +67,7 @@ deps = [
 run = """
 ../node_modules/.bin/esbuild src/cli.ts --bundle --platform=node --format=esm --outfile=$TARGET \
     --external:@jspm/generator --external:ora --external:picocolors --external:@babel/core \
-    --external:@jspm/plugin-rollup --external:node:*
+    --external:@jspm/plugin-rollup --external:rollup --external:@rollup/rollup-* --external:node:* --external:fsevents
 """
 
 [[task]]

--- a/cli/docs.md
+++ b/cli/docs.md
@@ -28,6 +28,8 @@ To customize the location of the output import map, the `--out` flag can be used
  
 ## Init
 
+▣  JSPM  -  Import Map Package Management
+
 **Usage**
   
 ```
@@ -58,6 +60,8 @@ jspm init ./my-project
 Initialize a project in the ./my-project directory.
 
 ## Ls
+
+▣  JSPM  -  Import Map Package Management
 
 **Usage**
   
@@ -124,6 +128,8 @@ List exports for the Lit package using the unpkg provider explicitly.
 
 ## Install
 
+▣  JSPM  -  Import Map Package Management
+
 **Usage**
   
 ```
@@ -174,6 +180,8 @@ Install packages into the import map tracing the package.json "exports" entry po
 jspm install
 ```
 ## Serve
+
+▣  JSPM  -  Import Map Package Management
 
 **Usage**
   
@@ -261,6 +269,8 @@ Start a server that does not generate the import map on startup, perform type st
 
 ## Build
 
+▣  JSPM  -  Import Map Package Management
+
 **Usage**
   
 ```
@@ -341,6 +351,8 @@ Build using a custom import map file.
 
 ## Publish
 
+▣  JSPM  -  Import Map Package Management
+
 **Usage**
   
 ```
@@ -351,7 +363,7 @@ Manages publishes to the JSPM providers, currently in experimental preview.
 WARNING: jspm publish is experimental and should only be used for prototyping.
 Unlike the https://ga.jspm.io/ CDN which is stable, reliability guarantees are
 not provided for publishing on https://jspm.io/. For reliable package delivery,
-use npm publish ΓÇö all npm packages are available on the https://ga.jspm.io/ CDN.
+use npm publish — all npm packages are available on the https://ga.jspm.io/ CDN.
 
 For publishing (default):
 
@@ -433,6 +445,8 @@ Download the application package foo@bar into the folder foo, merging its import
 
 ## Auth
 
+▣  JSPM  -  Import Map Package Management
+
 **Usage**
   
 ```
@@ -469,6 +483,8 @@ jspm auth
 List all available providers and their authentication status.
 
 ## Clear Cache
+
+▣  JSPM  -  Import Map Package Management
 
 **Usage**
   

--- a/cli/package.json
+++ b/cli/package.json
@@ -1,7 +1,7 @@
 {
   "name": "jspm",
   "type": "module",
-  "version": "4.5.0",
+  "version": "4.6.0",
   "description": "Import Map Package Manager",
   "license": "Apache-2.0",
   "bin": {
@@ -20,7 +20,8 @@
     "@jspm/generator": "^2.16.0",
     "@jspm/plugin-rollup": "^3.0.0",
     "ora": "^9.4.0",
-    "picocolors": "^1.1.1"
+    "picocolors": "^1.1.1",
+    "rollup": "^4.60.4"
   },
   "devDependencies": {
     "@antfu/eslint-config": "^9.0.0",
@@ -34,7 +35,6 @@
     "minimatch": "^10.0.1",
     "open": "^10.1.2",
     "prettier": "^3.8.3",
-    "rollup": "^4.60.4",
     "tinyspy": "^4.0.4",
     "typescript": "^5.9.3"
   }

